main.py

```diff
@@ -1,10 +1,50 @@
 import init_django_orm  # noqa: F401
+from json import load
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+
+    with open("players.json") as file:
+        players_dict = load(file)
+
+    for player_name, info in players_dict.items():
+        if not Race.objects.filter(
```

Comment:
u can make variables for info["guild"] and info["race"], so u can make ur code DRY